TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/ParsedHttpMessagesAsDicts.java

```diff
@@ -100,18 +102,16 @@ public static void fillStatusCodeMetrics(@NonNull IReplayContexts.ITupleHandling
         targetResponseOp.ifPresent(r -> context.setTargetStatus((Integer) r.get(STATUS_CODE_KEY)));
     }
 
-    private static byte[] getBytesFromByteBuf(ByteBuf buf) {
-        var bytes = new byte[buf.readableBytes()];
-        buf.getBytes(buf.readerIndex(), bytes);
-        return bytes;
-    }
-
     private static Map<String, Object> fillMap(LinkedHashMap<String, Object> map,
                                                HttpHeaders headers, ByteBuf content) {
-        String base64body = Base64.getEncoder().encodeToString(getBytesFromByteBuf(content));
-        map.put("body", base64body);
-        headers.entries().stream().forEach(kvp -> map.put(kvp.getKey(), kvp.getValue()));
-        return map;
+        try (var bufHolder = RefSafeHolder.create(Base64.encode(content, false, Base64Dialect.STANDARD))) {
```

Comment:
This is equivalent to the default [java.util.Base64.getEncoder()](https://docs.oracle.com/javase/8/docs/api/java/util/Base64.html) since Java8 which is the standard charset without line separators